ui/css/logviewer.css

```diff
@@ -62,6 +62,20 @@ body {
     vertical-align: middle;
 }
 
+.unknown-step-warning {
+    margin-bottom: 2px;
+    margin-right: 6px;
+    padding: 0 4px;
+    background-color: #fcf8e3;
+    border: 1px solid #fbd890;
+    border-radius: 2px;
+    color: #8a6d3b;
+}
+
+.step-title-unknown {
```

Comment:
So this basically nudges the text title downwards when the "unknown result step" span is visible so they are vertically aligned. Perhaps I should comment that in the css here. Not sure.